nrich-excel-api/src/main/java/net/croz/nrich/excel/api/request/CreateExcelReportRequest.java

```diff
@@ -48,4 +48,25 @@
      */
     private final MultiRowDataProvider multiRowDataProvider;
 
+    /**
+     * Creates {@link CreateExcelReportRequest} Builder instance from flat data.
```

Comment:
Please format this file, it should create an empty line here and in the same place for the other method.